jobs/config.json

```diff
@@ -9884,6 +9884,25 @@
       "sig-aws"
     ]
   },
+  "ci-kubernetes-e2e-kops-gce-ginkgodirect": {
+    "args": [
+      "--cluster=e2e-kops-gce-ginkgodirect.k8s.local",
+      "--deployment=kops",
+      "--env=KUBETEST_RUN_GINKGO_DIRECT=y",
```

Comment:
make that a flag?

Reply:
Happy to do that if you think so.  Here was my thought process though:

* Setting up the flags for ginkgo needs provider-specific logic (generally)  i.e. kops & kube-anywhere would have different flags.  The ginkgo-e2e.sh logic holds for kube-up, but not for kops, for example.
* I don't see a provider supporting multiple runners (both ginkgo direct and ginkgo-e2e.sh) long-term
* The env var I see as a temporary thing, until kops can get 100% to using ginkgo-direct.  I don't want to break the kops-on-AWS tests while I get it working though.
* Once it is working, I would just remove the env var.

I guess we could also just drop the flag in future though, but that's more of a breaking change...

Reply:
a flag is a lot less magical to understand :-)
for the --env flag here I need to know what reads this, how it converts to env, and what reads the env.
for a flag I just see the flag here and the corresponding definition in kubetest.

Reply:
I agree. We're trying to unify and clean-up the flags in kubetest.